erroranalysis/erroranalysis/version.py

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) Microsoft Corporation
+# Licensed under the MIT License.
+
+name = 'erroranalysis'
```

Comment:
do you mind change this to match raiwidget?
So all packages share same version for better control

Reply:
discussed, this will be done for raianalysis